samples/boards/nordic/system_off/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,22 @@
+/ {
+	sram0@2007e000 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		reg = <0x2007e000 DT_SIZE_K(4)>;
+		zephyr,memory-region = "RetainedMem";
+		status = "okay";
+
+		retainedmem0: retainedmem {
+			compatible = "zephyr,retained-ram";
+			status = "okay";
+		};
+	};
+
+	aliases {
+		retainedmemdevice = &retainedmem0;
+	};
+};
+
+&sram0 {
```

Comment:
Shouldn't it be rather `sram0_image` (chosen as `zephyr,sram`), the one to be shrunk?

Reply:
yes, it should. I have corrected it now. Additionally, I have noticed that last 64 kB of RAM is assigned to IPC buffers, so I have moved the `retained_mem` region accordingly

Reply:
I think it would be good to apply same corrections in [tests/drivers/retained_mem/api/boards/nrf5340dk_nrf5340_cpuapp.overlay](https://github.com/zephyrproject-rtos/zephyr/pull/84494/files#diff-ba8a4e78adadf0568cb95e4ef5250c3a07bd92e66a8260c7230d8cedc5fcc98b). It's just a test, but still.

Reply:
done